packages/mutation-testing-elements/src/components/mutation-test-report-totals.ts

```diff
@@ -155,11 +160,11 @@ export class MutationTestReportTotalsComponent extends LitElement {
         <div class="progress">
           <div class="progress-bar bg-${coloringClass}" role="progressbar" aria-valuenow="${mutationScoreRounded}"
             aria-valuemin="0" aria-valuemax="100" .style="${progressBarStyle}">
-            ${mutationScoreRounded}%
+            ${mutationScoreRounded ? mutationScoreRounded + '%' : ''}
           </div>
         </div>
       </td>
-      <th style="width: 50px;" class="no-border-left text-center text-${coloringClass}">${mutationScoreRounded}</th>
+      <th style="width: 50px;" class="no-border-left text-center text-${coloringClass}">${mutationScoreRounded ? mutationScoreRounded : 'N/A'}</th>
```

Comment:
What happens when `mutationScore` is `0`? That's a falsy value, so probably will be `n/a`. Shouldn't be, right?